packages/containers/Team/createColumns.js

```diff
@@ -38,19 +38,22 @@ const MenuWrapper = styled.div`
   position: absolute;
   width: 100%;
   min-height: 42px;
-  border: 1px solid ${({ theme }) => theme.color.gallery};
+  background-color: ${({ theme }) => theme.color.emperor}!important;
```

Comment:
does not it really work without important?
Important is possible to use with a library styles.

Reply:
Without important the tooltip background will be overrided by react-power-tooltip and dark arrow will appear